src/layouts/error.vue

```diff
@@ -17,11 +17,11 @@
               <a
                 class="btn btn--outlined"
                 :href="getOAuthLoginAPI()"
-                @click="onClickLogEvent('Register', 'RegisterSignIn', 'RegisterSignIn(error)', 1)"
+                @click="onClickLogEvent('Register', 'RegisterSignIn', 'RegisterSignIn(error page)', 1)"
               >{{ $t('signIn') }}</a><a
                 class="btn btn--outlined"
                 :href="getOAuthLoginAPI()"
-                @click="onClickLogEvent('Register', 'RegistersignUp', 'RegisterSignUp(error)', 1)"
+                @click="onClickLogEvent('Register', 'RegistersignUp', 'RegisterSignUp(error page)', 1)"
```

Comment:
Is logging values containing space a concern? (e.g. inconvenient for query?)

Reply:
not rly, just length limit